docs/src/main/sphinx/connector/mysql.rst

```diff
@@ -80,6 +80,30 @@ property, which can be set to ``UNNECESSARY`` (the default),
 
 .. include:: jdbc-type-mapping.fragment
 
+.. _mysql-sql-support:
+
+SQL support
+-----------
+
+The connector provides read access and write access to data and metadata in the
+MySQL database. In addition to the :ref:`globally available <sql-globally-available>` and
+:ref:`read operation <sql-read-operations>` statements, the connector supports
+the following features:
+
+* :doc:`/sql/insert`
+* :doc:`/sql/delete`
+* :ref:`sql-schema-table-management`
```

Comment:
Notably ALTER SCHEMA isn't supported from the ones under `sql-schema-table-management`.

Rest of information looks correct.

Reply:
@Jessie212 : Per https://github.com/trinodb/trino/pull/8669#discussion_r677552792 linking to "SQL schema table management" isn't correct because only some of the linked statements are supported.

Replace "* :ref:`sql-schema-table-management`" with:

* :doc:`/sql/create-table`
* :doc:`/sql/create-table-as`
* :doc:`/sql/drop-table`
* :doc:`/sql/create-schema`
* :doc:`/sql/drop-schema`
* :doc:`/sql/comment`